### PR TITLE
Fix bug blocking transfer of context from Crowdin

### LINKED ISSUE #7916: Context not transferred from Crowdin
### Observed behavior

Initially noted by @radinamatic 

This [Vue SFC](https://github.com/learningequality/kolibri/blob/release-v0.14.x/kolibri/plugins/learn/assets/src/views/classes/ClassAssignmentsPage.vue#L68) did not have the [context for its translation](https://crowdin.com/translate/kolibri/3798/en-es?filter=basic&value=0#275962) transferred into the code.

This issue should be fixed in the context of 0.15. This work may ultimately be blocked by some necessary work in  https://github.com/learningequality/kolibri/issues/7709 

### Expected behavior

`yarn transfercontext` should transfer all context from Crowdin into the components associated with the translation strings.

### User-facing consequences

Internal facing only - but context will not be available in the code, which can result in losing context when uploading the next round of translations. 

### Context

Kolibri 0.15

### PR DESCRIPTION
**Pull request by nucleogenesis**
## Summary

The issue ultimately was that I initially relied on the ACH language. This is a pseudolanguage and, as such, it seems that the context does not properly come through the CSV as it does in other languages. It seems that this might have been a result of a change on Crowdin's side as the initial context transfers seemed to go as well as expected.

Also - `make i18n-download` failed on some stuff that I should have taken care of when I moved the i18n into `kolibri-tools`.

One of those is dealing with fonts - which IIRC are 100% only to do with Kolibri. I made it so that it doesn't fail - but the font regeneration will still only work in Kolibri.

Context transfer now works and, although I haven't tested it, should also work smoothly in Studio as well when the `CROWDIN_PROJECT` env var is set to `contentcuration`.

## References

Fixes #7916 

## Reviewer guidance

Be sure you have your `CROWDIN_API_KEY` env var set then checkout develop and run `make i18n-download branch=release-v0.14.x`. Hit `y` to continue anyway both times.

Once the files are all downloaded and you see the message that context has indeed transferred, you are all set!

----

## Testing checklist

- [x] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

## PR process

- [ ] PR has the correct target branch and milestone
- [ ] PR has 'needs review' or 'work-in-progress' label
- [ ] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

## Reviewer checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

File: packages/kolibri-tools/lib/i18n/intl_code_gen.js [hunk 1]
```diff
@@ -123,11 +119,7 @@ const intlFooter = `
 const intlModule =
   commonHeader + intlHeader + languageInfo.map(generateIntlItems).join('') + intlFooter;
 
-const intlModulePath = path.resolve(
-  __dirname,
-  '../../kolibri/core/assets/src/utils/intl-locale-data.js'
-);
-
+const intlModulePath = path.resolve('./intl-locale-data.js');
```
Comment (rtibbles) [inline packages/kolibri-tools/lib/i18n/intl_code_gen.js]: This seems to have resulted in the intl-locale-data file being put in the root of the repo - this seems like the wrong place.

Comment (indirectlylit) [reply packages/kolibri-tools/lib/i18n/intl_code_gen.js]: yes, `intl-locale-data.js` and `vue-intl-locale-data.js` show as new rather than updated files

**Comment by radinamatic** [comment]
@nucleogenesis Is this a new Crowdin requirement, I already have the `CROWDIN_API_KEY` set up?

```
(kolibri3) didivm@didivm-VirtualBox:~/LE/kolibri$ make i18n-download branch=release-v0.14.x
python packages/kolibri-tools/lib/i18n/crowdin.py rebuild-translations release-v0.14.x
ERROR: 
Please set the `CROWDIN_LOGIN` environment variable to your Crowdin username.

Makefile:215: recipe for target 'i18n-download-translations' failed
make: *** [i18n-download-translations] Error 1
(kolibri3) didivm@didivm-VirtualBox:~/LE/kolibri$ 
```

**Comment by radinamatic** [comment]
Disregard the above, all good now!

Tested locally and the missing context values are transferred, yey! 💯 

`'i18n-regenerate-fonts'` did fail, but was successful after running `make i18n-download-source-fonts`, and in the end only the linter was failing, which if I remember correctly was issue even before...?

1 | 2
--- | ---
LEDev2104 (start)  Running  - Oracle VM VirtualBox_003 | LEDev2104 (start)  Running  - Oracle VM VirtualBox_004

**Review by radinamatic — APPROVED** [review]
Looking & working good for me! 👍🏽

**Pull request merged**